compose/docker-compose.override-lnd.yml

```diff
@@ -13,7 +13,7 @@ services:
       - ${LND_DATA:?}:/root/.lnd
       - ${LND_CONF:?}:/root/.lnd/lnd.conf
       - ${LND_AUTOUNLOCK_PWD:?}:/tmp/pwd:ro
-    command: lnd
+    entrypoint: lnd
```

Comment:
`command:` contacts to the entry-point of the image, resulting in a command sent to the image like:
 ```
lnd lnd
```
I could have work just removing the line, but I feel explicitly including it makes to have a little bit of better control